pkgs/by-name/ey/eyewitness/package.nix

```diff
@@ -0,0 +1,66 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, python3Packages
+, cmake
+, xvfb-run
+, firefox-esr
+, geckodriver
+, makeWrapper
+}:
+
+stdenv.mkDerivation rec {
```

Comment:
Please use python3Packages.buildPythonApplication. See the nixpkgs manual.

Reply:
specified pyproject = false;